<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Theoretical Diary</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="MessageWidget" name="messageWidget" native="true"/>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="pageMainMenu">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="MainMenu" name="mainMenu" native="true"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageDiaryTabWidget">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="DiaryTabWidget" name="diaryTabWidget">
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="documentMode">
           <bool>true</bool>
          </property>
          <property name="tabsClosable">
           <bool>true</bool>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
          <property name="tabBarAutoHide">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageSettings"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDiary">
    <property name="title">
     <string>Diary</string>
    </property>
    <widget class="QMenu" name="menuOpenRecentDiary">
     <property name="title">
      <string>Recent Diaries</string>
     </property>
    </widget>
    <addaction name="actionNewDiary"/>
    <addaction name="actionOpenDiary"/>
    <addaction name="menuOpenRecentDiary"/>
    <addaction name="actionSaveDiary"/>
    <addaction name="actionSaveDiaryAs"/>
    <addaction name="actionImportDiary"/>
    <addaction name="actionExportDiary"/>
    <addaction name="separator"/>
    <addaction name="actionLockDiary"/>
    <addaction name="separator"/>
    <addaction name="actionUploadDiary"/>
    <addaction name="actionDownloadDiary"/>
    <addaction name="actionConnectGoogleDrive"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEntry">
    <property name="title">
     <string>Entry</string>
    </property>
    <addaction name="actionEditEntry"/>
    <addaction name="actionDeleteEntry"/>
    <addaction name="actionJumpToToday"/>
    <addaction name="actionSearchEntries"/>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Preferences</string>
    </property>
    <widget class="QMenu" name="menuTheme">
     <property name="title">
      <string>Theme</string>
     </property>
     <addaction name="actionLightTheme"/>
     <addaction name="actionDarkTheme"/>
    </widget>
    <addaction name="menuTheme"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionReportABug"/>
    <addaction name="actionRequestAFeature"/>
   </widget>
   <addaction name="menuDiary"/>
   <addaction name="menuEntry"/>
   <addaction name="menuPreferences"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="SaveEntry">
   <property name="text">
    <string>Save Entry</string>
   </property>
  </action>
  <action name="actionDeleteEntry">
   <property name="text">
    <string>Delete Entry</string>
   </property>
   <property name="shortcut">
    <string>Shift+Del</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionReportABug">
   <property name="text">
    <string>Report a Bug</string>
   </property>
  </action>
  <action name="actionRequestAFeature">
   <property name="text">
    <string>Request a Feature</string>
   </property>
  </action>
  <action name="actionClearHistory">
   <property name="text">
    <string>Clear History</string>
   </property>
  </action>
  <action name="actionLockDiary">
   <property name="text">
    <string>Lock Diary</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionUploadDiary">
   <property name="text">
    <string>Upload Diary</string>
   </property>
  </action>
  <action name="actionDownloadDiary">
   <property name="text">
    <string>Download Diary</string>
   </property>
  </action>
  <action name="actionConnectGoogleDrive">
   <property name="text">
    <string>Connect Google Drive</string>
   </property>
  </action>
  <action name="actionJumpToToday">
   <property name="text">
    <string>Jump to Today</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionSearchEntries">
   <property name="text">
    <string>Search Entries</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actionLightTheme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionDarkTheme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionOpenDiary">
   <property name="text">
    <string>Open Diary...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNewDiary">
   <property name="text">
    <string>New Diary...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionImportDiary">
   <property name="text">
    <string>Import Diary...</string>
   </property>
  </action>
  <action name="actionExportDiary">
   <property name="text">
    <string>Export Diary</string>
   </property>
  </action>
  <action name="actionSaveDiary">
   <property name="text">
    <string>Save Diary</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveDiaryAs">
   <property name="text">
    <string>Save Diary As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionEditEntry">
   <property name="text">
    <string>Edit Entry</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MessageWidget</class>
   <extends>QWidget</extends>
   <header>gui/MessageWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MainMenu</class>
   <extends>QWidget</extends>
   <header>src/gui/MainMenu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DiaryTabWidget</class>
   <extends>QTabWidget</extends>
   <header>src/gui/DiaryTabWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
